cf_remote/main.py

```diff
@@ -313,7 +320,7 @@ def get_cloud_hosts(name, private_ips=False):
 
     ret = []
     for host in hosts:
-        if private_ips:
+        if private_ips and "private_ips" in host:
```

Comment:
Why is this necessary? There's `host.get(key, [])` below so a request to get private IPs will just return empty lists for hosts that don't have private IPs. This change here twists the logic.

Reply:
@vpodzime Yes, this changes the logic to fall back to public IPs if private IPs don't exist. This function is called with `private_ips = True` when we are looking for what IP to use for bootstrapping. In the case of saved hosts, for example from DigitalOcean, there is no private IP, so falling back to public is correct.

Reply:
Sure, I can do that.

Reply:
Added in last commit.